src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
```

Comment:
The checklist specifies not to use HashMaps or any other Map data structure for this task. Try to solve the problem without using these data structures.